src/test/java/RecipeTest.java

```diff
@@ -58,9 +62,8 @@ public void addBeginningStep() {
         new Step("Put the cheese in the bread")
     );
 
-    Recipe expected = new Recipe(NAME, DESCRIPTION, expectedSteps);
     recipe.addStep(0, new Step("Index 0!"));
```

Comment:
Will this work or not? I took a look at recipe.java, and the check is position >= 0; if someone wants to add a step at position 0, will it just add it to the top?

Reply:
I think so, the tests all passed when I ran them

Reply:
Should we enforce non-0 indexing? When you're inserting a recipe step? It seems kind of odd for general users.

Reply:
I don't think a user will ever directly invoke this method, because it'll have to go through JavaScript first. My idea was that the JavaScript can handle indexing issues (ie, user wants to add a step 1, which JavaScript changes to index 0), and then call this Java method.

Reply:
okok